web/Makefile

```diff
@@ -11,12 +11,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: blob
+include ../Makefile.INCLUDE
 
-blob:
-	$(MAKE) -C blob
+all: blob/files.go
+
+SUFFIXES:
+
+# Do not embed map files and the non-minified bootstrap files.
```

Comment:
The note should actually explain it better. Like "embed-static.sh does not include ..."